R/event.xyplot.R

```diff
@@ -55,3 +121,6 @@ event.xyplot.formula <-
     foo$call <- sys.call(sys.parent())
     foo
   }
+
+#' @import utils
+utils::globalVariables(c("ind"))
```

Comment:
This one's tricky - ind shouldn't be a global variable, but I'm not sure how to deal with it.
I'm happy to leave this as is if we just add an issue for it to be addressed later.

Reply:
No worries. Again, I set it as a global variable because R CMD check complained:

```
 event.xyplot.formula : <anonymous>: no visible binding for global
  variable ‘ind’
```

I'll raise an issue for posterity.

Reply:
Is now in #145